### PR TITLE
feat: migrate sort popover to mantine

### PR DESCRIPTION
**Pull request by IrakliJani**
closes: https://github.com/lightdash/lightdash/issues/5918

CleanShot 2023-06-15 at 14 11 44@2x

### CODE REVIEW

**Review by PriPatel — CHANGES_REQUESTED** [review]
Wahoooo - this looks so much nicer!!

Only comment is on the drag handles - they look 'horizontal' when they should be 'vertical' (like the current way attached?) 

Screenshot 2023-06-15 at 12 41 10

**Comment by IrakliJani** [comment]
@PriPatel updated

CleanShot 2023-06-15 at 15 58 23@2x

File: packages/frontend/src/components/LimitButton/index.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const LimitButton: FC<Props> = memo(
             <Popover
                 disabled={disabled}
                 opened={opened}
-                position="bottom"
+                position="top"
```
Comment (IrakliJani) [inline packages/frontend/src/components/LimitButton/index.tsx:31]: why top? good question. popover does not overlap results table, you can still see headers and data, find it a better UX especially for sorting popover

Comment (joaoviana) [reply packages/frontend/src/components/LimitButton/index.tsx:31]: agree!

**Review by yasmin-e — APPROVED** [review]

**Pull request merged**